Sources/AtlasConnectivity/main.swift

```diff
@@ -1,11 +1,8 @@
 import Foundation
 import MongoSwiftSync
 
-private let configs = ["ATLAS_REPL", "ATLAS_SHRD", "ATLAS_FREE", "ATLAS_TLS11", "ATLAS_TLS12"]
+private let configs = ["ATLAS_REPL", "ATLAS_SHRD", "ATLAS_FREE", "ATLAS_TLS11", "ATLAS_TLS12", "ATLAS_SERVERLESS"]
 private let srvConfigs = configs.map { $0 + "_SRV" }
-/// Currently, almost all of the Atlas test instances are running 3.4 which does not support the new "hello" command.
```

Comment:
these instances were recently all upgraded to 3.6 (and new enough patch versions to have "hello" backported), so the special-casing is no longer necessary